syntax = "proto3";

package skaimsg.feetpos;

message FeetPosProtoMsg {
    message CameraFrame {
        // camera mac address turned into serial num
        uint64 camera_id = 1;

        // 1e9 * epoch timestamp
        uint64 timestamp = 2;

        message FeetPosInFrame {
            message xyz {
                float x = 1;
                float y = 2;
                float z = 3;
            }
            repeated xyz feetpos = 1;
        }
        repeated FeetPosInFrame people_in_frame = 3;
    }
    repeated CameraFrame camera_frames = 1;
}